config/pq_capablities.ini

```diff
@@ -259,25 +259,29 @@ platformsupport = true
 
 [DimmingMode]
 range  = local,fixed
+index  = 1,0
```

Comment:
index is very generic word and can not relate what it relates to without checking code. Should we provide some documentation change for this?

Reply:
Chages approved